### ISSUE TITLE
Password required, but pw not accepted

### ISSUE THREAD

**Issue by nephilim75** [nextcloud/server#2863]
### Steps to reproduce
1. upgrading/ installing NC 11 and trying to change AJAX to CRON or trying to add apps

### Expected behaviour
If the admin password is entered, the password should be accepted

### Actual behaviour
It is asking for a password but doesn't accept the given (admin) password.

### Server configuration
**Operating system**: Debian 8 (Jessie)

**Web server**:nginx/1.6.2

**Database:** MySQL

**PHP version**: 5.6.27

**Nextcloud version:** 11.0.0

**Updated from an older Nextcloud/ownCloud or fresh install:** upgraded from 10.0.2 AND fresh installation

**Where did you install Nextcloud from:** nextcloud.com/install

**Signing status:** No errors have been found.
<details>
<summary>Signing status</summary>

**List of activated apps:** standard, no extra apps installed or activated due to described issue

> Activated:
>   - activity: 2.4.1
>   - comments: 1.1.0
>   - dav: 1.1.1
>   - federatedfilesharing: 1.1.1
>   - federation: 1.1.1
>   - files: 1.6.1
>   - files_pdfviewer: 1.0.1
>   - files_sharing: 1.1.1
>   - files_texteditor: 2.2
>   - files_trashbin: 1.1.0
>   - files_versions: 1.4.0
>   - files_videoplayer: 1.0.0
>   - firstrunwizard: 2.0
>   - gallery: 16.0.0
>   - logreader: 2.0.0
>   - lookup_server_connector: 1.0.0
>   - nextcloud_announcements: 1.0
>   - notifications: 1.0.1
>   - password_policy: 1.1.0
>   - provisioning_api: 1.1.0
>   - serverinfo: 1.1.1
>   - sharebymail: 1.0.1
>   - survey_client: 0.1.5
>   - systemtags: 1.1.3
>   - theming: 1.1.1
>   - twofactor_backupcodes: 1.0.0
>   - updatenotification: 1.1.1
>   - workflowengine: 1.1.1
> Disabled:
>   - admin_audit
>   - encryption
>   - external
>   - files_accesscontrol
>   - files_automatedtagging
>   - files_external
>   - files_retention
>   - templateeditor
>   - user_external
>   - user_ldap
>   - user_saml

**The content of config/config.php:**
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "ocdq5v67agmj",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "sub.domain.com"
        ],
        "datadirectory": "\/media\/usbhdd\/nextcloud",
        "overwrite.cli.url": "https:\/\/sub.domain.com",
        "dbtype": "mysql",
        "version": "11.0.0.10",
        "dbname": "nextcloud",
        "dbhost": "localhost",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "memcache.local": "\\OC\\Memcache\\APC"
    }
}

```

**Are you using external storage, if yes which one:** /media/usbhdd/nextcloud

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** No

#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:** Anyone

**Operating system:**

#### Web server error log
<details>
<summary>Web server error log</summary>
```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>
```
It is empty
```
</details>

Kind regards
//neph

**Comment by enoch85** [comment]
@nephilim75 Have you tried to reset the admin password to something else and then try again?

**Comment by nephilim75** [comment]
Unfortunately the same behaviour. It is still asking for a legitimation by password. Once it is entered, it is asking again and is doing nothing obviously. No error message is coming up.
-- 
Von (m)einer Couch gesendet...

Am 27. Dezember 2016 19:03:04 MEZ, schrieb Daniel Hansson <notifications@github.com>:
>@nephilim75 Have you tried to reset the admin password to something
>else and then try again?

**Comment by lsascha** [comment]
Hello, i have the exact same issue.

I upgraded from owncloud 8 all the way up to nextcloud 11 as suggested (with every upgrade step between). So far it seems everything worked perfectly, except that i can't install any app because the password request pops up every time. Entering the password that should be correct does nothing and the window pops up again. Entering the wrong password does show a small error at the top of the browser window that the password was wrong, so i am pretty sure its not the password.

SOLUTION:
while i was writing i checked for any javascript errors or something and noticed it returns a timestamp.
After checking it i noticed that the timestamp was wrong, so i checked the server time and noticed the server-time was all wrong.

I synced the server-time and BAM. everything works now. :)
